tensilelite/Tensile/ClientWriter.py

```diff
@@ -111,7 +111,13 @@ def main( config ):
     functionNames.append("tensile_%s" % (problemType))
     problemSizes = ProblemSizesMock(exactLogic)
     biasTypeArgs   = BiasTypeArgs(problemType, [problemType["DataType"]])
-    activationArgs = ActivationArgs(problemType, [[{'Enum': 'relu'}]]) if problemType["ActivationType"] == 'all' else ""
+    activationEnums = [[{'Enum': 'relu'}]]
```

Comment:
Could you add your example here to let user know how to set different activation type in LibraryClient stage?